lib/auth/grpcserver.go

```diff
@@ -4972,6 +4973,11 @@ func (g *GRPCServer) DeleteAllUserGroups(ctx context.Context, _ *emptypb.Empty)
 	return &emptypb.Empty{}, trace.Wrap(auth.DeleteAllUserGroups(ctx))
 }
 
+// GetBackend returns the backend from the
```

Comment:
Seems the comment is an unfinished sentence?

Reply:
Guh, whoops. Thanks!

Reply:
Annoyingly, looks like I just forgot to save the file in the editor before committing, heh. Anyway, fixed!